magstock/templates/regextra.html

```diff
@@ -41,8 +43,8 @@
         if ($.field('badge_printed_name')) {
             $.field('badge_printed_name').parents('.form-group').remove();
         }
-        {% if c.PAGE_PATH != '/registration/form' %}
-            $.field('amount_extra').parents('.form-group').hide();
+        {% if c.PAGE_PATH == '/registration/form' %}
+            $('#vehicle-warning').hide();
```

Comment:
Seems okay, but it would probably be better to just not render the vehicle warning on the admin page in the first place.

The main reason why we do the dynamic Javascript hiding most of the time is that we may be doing it in a plugin, and the rendering is happening in a different template we don't have access to.  But in this case, the vehicle warning block is defined in this very same file (https://github.com/magfest/magstock/blob/751fb593b519c5398bf8d88f344cd724a52710e9/magstock/templates/regextra.html#L106), so putting this dynamic if statement in a different place makes the code a little harder to follow.